src/test/java/seedu/duke/logic/commands/BusCommandTest.java

```diff
@@ -0,0 +1,22 @@
+package seedu.duke.logic.commands;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import seedu.duke.exceptions.CustomException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
can this import be removed?
```suggestion
```

Reply:
I've removed this file. Wasn't meant to commit it to branch

Reply:
Okie! Yah, I realized it wasn't running!